### PR TITLE
add getting started guides directory

### PR DESCRIPTION
**Pull request by reese-lee**

### CODE REVIEW

File: getting-started-guides/.gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.idea/
```
Comment (jack-berg) [inline getting-started-guides/.gitignore:1]: Can delete this.

File: getting-started-guides/.gitignore [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+# .NET build artifacts
+[Bb]in/
+[Oo]bj/
+
+# Java build artifacts
+build/
+.gradle/
```
Comment (jack-berg) [inline getting-started-guides/.gitignore:9]: Let's reorganize to this directory structure:

-> getting-started-guides/
  --> java/
    --> instrumented/
    --> uninstrumented/
  --> dotnet/
    --> instrumented/
    --> uninstrumented/

**Review by jack-berg — APPROVED** [review]

**Comment by jack-berg** [comment]
@alanwest take a look, LGTM.

**Review by alanwest — APPROVED** [review]
:shipit:

**Pull request merged**